### DISCUSSION TITLE
想问下kitty中的这个显示效果如何在其他地方中复现

### DISCUSSION THREAD

**Discussion by Looouiis** [subframe7536/maple-font#266] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/afae49ee-c303-4a8f-b309-f138a637c98f)
如图所示，如何让多个#实现前面几个是小号最后一个正常大小呀，感觉这样挺好看的。我尝试只能在kitty终端模拟器中看到这个效果，换wezterm等其他就没有这个效果了，想问下这个效果是可以手动开启的嘛，应该如何开启呀
我的环境是archlinux，安装了ttf-maple-sc-nerd
![image](https://github.com/user-attachments/assets/a11b8529-3fac-4a22-a603-8e374bea8439)

**Comment by subframe7536** [comment, marked as answer]
#141

这个其实是bug...

**Reply by Looouiis** [reply]
好吧😭我前面感觉这个做tui播放器的进度条正好合适呢
